index.js

```diff
@@ -119,7 +119,7 @@ const getTextVariation = (color, opacity = 1, threshold = .6) => {
  * when getting lighter, and mixing black gives a smooth smokey color for darkness. We can't increase saturation
  *because that unintentionally colors grays.
  */
-const coloredBackgroundAdjust = (colorString, whitenessAdjustment) => {
+const adjustLightness = (colorString, whitenessAdjustment) => {
```

Comment:
I had gotten an error trying to NPM link the select project because of this but let me know if I should change this back.

Reply:
@derekstory This is the name of it in master: https://github.com/SpiderStrategies/scoreboard-colors/blob/master/index.js#L124

It's probably cleaner if you rebase so `v6-updated-color-palette` has the latest from master.